src/cmd_parser.c

```diff
@@ -318,7 +323,11 @@ check_and_set_compression_type(const char *type_str,
         *type = CR_CW_BZ2_COMPRESSION;
     } else if (!strcmp(compress_str->str, "xz")) {
         *type = CR_CW_XZ_COMPRESSION;
-    } else if (!strcmp(compress_str->str, "zstd")) {
```

Comment:
This would be a breaking change so we cannot do that. 
I am not sure if the intention was to specify the compression by its suffix but I would keep it as `zstd`.